src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -83,7 +84,7 @@ const PastAnalysisTable = () => {
         analysis.dcmImage.StudyDescription,
         analysis.dcmImage.PatientID,
         analysis.dcmImage.PatientBirthDate,
-        `${analysis.dcmImage.PatientAge}`,
+        `${Utils.calculatePatientAge(analysis.dcmImage.PatientBirthDate)}`,
```

Comment:
I would suggest to either

a) Remove string interpolation since there is no need to make it into a string
```suggestion
        Utils.calculatePatientAge(analysis.dcmImage.PatientBirthDate),
```

OR

b) Add `y` at the end of the string so its consistent with the "Create Analysis" page where we have `y` right after the age number.
```suggestion
        `${Utils.calculatePatientAge(analysis.dcmImage.PatientBirthDate)}y`,
```